### PR TITLE
gpu_operator_run_gpu-burn: make gpu-burn execution easier to reproduce

### PR DESCRIPTION
**Pull request by kpouget**
This PR slightly rewrites the GPU Burn role to make it easier to re-execute outside of the CI, by storing the required ConfigMap definition into files and simplifying a bit the Pod spec.
```
008__gpu-operator__run_gpu_burn/000_configmap_gpu-burn_entrypoint.yml
008__gpu-operator__run_gpu_burn/000_configmap_gpu-burn_src.yml
008__gpu-operator__run_gpu_burn/001_pod_gpu-burn_dgxa100.yml
```

### CODE REVIEW

File: roles/gpu_operator_run_gpu-burn/tasks/main.yml [hunk 1]
```diff
@@ -18,20 +18,32 @@
...
+       --dry-run=client
+       -oyaml
+       > {{ artifact_extra_logs_dir }}/000_configmap_gpu-burn_entrypoint.yml
 
 - name: Create the src ConfigMap
   command:
+    oc apply -f {{ artifact_extra_logs_dir }}/000_configmap_gpu-burn_entrypoint.yml
+
+# Store the configmap into a file, for easy reproduction from artifacts
+- name: Create the src ConfigMap
```
Comment (omertuc) [inline roles/gpu_operator_run_gpu-burn/tasks/main.yml:36]: The title "Create the src ConfigMa"p" appears 3 times

Comment (kpouget) [reply roles/gpu_operator_run_gpu-burn/tasks/main.yml:36]: ack, I'll push the fix at the end of the current test

**Comment by kpouget** [comment]
test succeeded in the [previous run](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/300/pull-ci-openshift-psap-ci-artifacts-master-gpu-operator-e2e/1468899616853331968/artifacts/gpu-operator-e2e/presubmit-master/artifacts/010__gpu_operator__run_gpu_burn/), I only pushed minor updates, no need to restest

/approve
/override ci/prow/gpu-operator-e2e

**Comment by kpouget** [comment]
thanks @omertuc :)

**Pull request merged**